lib/error/error.cpp

```diff
@@ -75,7 +76,8 @@ std::string severity_to_string(const Severity& s) {
     case Severity::High:
         return "High";
     }
-    throw std::out_of_range("No known string conversion for provided enum of type Severity.");
+    EVLOG_error << "No known string conversion for provided enum of type Severity:" << s << ". Defaulting to High.";
```

Comment:
`Severity s` is not a string
```suggestion
    EVLOG_error << "No known string conversion for provided enum of type Severity. Defaulting to High.";
```

.. same for the following enums